source/hud/BlockCursor.cpp

```diff
@@ -71,8 +71,8 @@ void BlockCursor::onEvent(const SDL_Event &event, const Hotbar &hotbar) {
 			const Block &block = Registry::getInstance().getBlock(blockId);
 			const Item &item = Registry::getInstance().getItem(hotbar.currentItem());
 
-			if (block.id() && !block.onBlockActivated(m_selectedBlock, m_player, m_world)
-			&& hotbar.currentItem() && item.isBlock()) {
+			if (block.id() && !block.onBlockActivated({m_selectedBlock.x,m_selectedBlock.y,m_selectedBlock.z}, m_player, m_world)
```

Comment:
Why did you have to change this?
Also, please add a whitespace after each comma (check other files to understand code standard since I haven't written a note in the wiki about this yet)